src/asm/rgbasm.1

```diff
@@ -7,89 +7,187 @@
 .\"
 .Dd July 8, 2019
 .Dt RGBASM 1
-.Os RGBDS Manual
+.Os
 .Sh NAME
 .Nm rgbasm
 .Nd Game Boy assembler
 .Sh SYNOPSIS
-.Nm rgbasm
-.Op Fl EhLVvw
+.Nm
 .Op Fl b Ar chars
 .Op Fl D Ar name Ns Op = Ns Ar value
+.Op Fl E
 .Op Fl g Ar chars
+.Op Fl h
 .Op Fl i Ar path
+.Op Fl L
 .Op Fl M Ar dependfile
 .Op Fl o Ar outfile
 .Op Fl p Ar pad_value
 .Op Fl r Ar recursion_depth
-.Ar file
+.Op Fl V
+.Op Fl v
+.Op Fl w
+.Ar
 .Sh DESCRIPTION
 The
 .Nm
-program creates an object file from an assembly source file.
+program creates a RGB object file from an assembly source file.
 The input
 .Ar file
 can be a file path, or
 .Cm \-
 denoting
 .Cm stdin .
-Its arguments are as follows:
+.Pp
+Note that options can be abbreviated as long as the abbreviation is unambiguous:
+.Fl -verb
+is
+.Fl -verbose ,
+but
+.Fl -ver
+could also be
+.Fl -version .
+The arguments are as follows:
 .Bl -tag -width Ds
-.It Fl b Ar chars
+.It Fl b , Fl -binary-digits Ar chars
 Change the two characters used for binary constants.
 The defaults are 01.
-.It Fl D Ar name Ns Op = Ns Ar value
+.It Fl D , Fl -define Ar name Ns Oo = Ns Ar value Oc
 Add string symbol to the compiled source code. This is equivalent to
 .Ar name
 .Cm EQUS
 .Qq Ar "value"
 in code. If a value is not specified, a value of 1 is given.
-.It Fl E
+.It Fl E , Fl -export-all
 Export all labels, including unreferenced and local labels.
-.It Fl g Ar chars
-Change the four characters used for binary constants.
+.It Fl g , Fl -gfx-chars Ar chars
+Change the four characters used for gfx constants.
 The defaults are 0123.
-.It Fl h
+.It Fl h , Fl -halt-without-nop
 By default,
 .Nm
 inserts a
-.Sq nop
+.Cm nop
 instruction immediately after any
-.Sq halt
+.Cm halt
```

Comment:
`Ic`, not `Cm`. (I may have confused them when I first wrote the manpage though.)

`Cm` is for fixed strings that are predefined arguments to a command. For example, `who(1)` can be called as “who am i”; [“am i” is marked up with `Cm`](https://github.com/openbsd/src/commit/9e053c74bf9a37baa0da7c36e556667226b2df79).

Reply:
Okay; I didn't find the documentation very clear about this. Will fix as suggested. (Should I check for more places, or is this the only one?)

Reply:
I didn’t search for others.